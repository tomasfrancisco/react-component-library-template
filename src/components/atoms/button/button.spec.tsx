import { render } from "@testing-library/react";
import React from "react";
import { Button } from ".";

describe("button", () => {
  it("renders as expected", () => {
    const { container } = render(<Button>Test</Button>);

    expect(container).toMatchSnapshot();
  });
});
