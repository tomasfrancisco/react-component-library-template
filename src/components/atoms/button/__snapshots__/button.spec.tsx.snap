// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button renders as expected 1`] = `
<div>
  <button
    class="button__StyledButton-sc-lhg3uz-0 htEsLP"
  >
    Test
  </button>
</div>
`;
